src/tinkerio.py

```diff
@@ -314,14 +314,18 @@ def __init__(self, name="tinker", **kwargs):
         self.warn_vn = False
         super(TINKER,self).__init__(name=name, **kwargs)
 
+    def CheckEnvironmentTinker(self):
+        tinkerpath = os.getenv("TINKERPATH", default="")
+        return tinkerpath
+
     def setopts(self, **kwargs):
         
         """ Called by __init__ ; Set TINKER-specific options. """
 
         ## The directory containing TINKER executables (e.g. dynamic)
         if 'tinkerpath' in kwargs:
             self.tinkerpath = kwargs['tinkerpath']
-            if not os.path.exists(os.path.join(self.tinkerpath,"dynamic")):
+            if not os.path.exists(os.path.join(self.tinkerpath,"dynamic")) and 'dynamic' not in os.environ.keys():
```

Comment:
Could you comment on this? Is "dynamic" supposed to be an environment variable too?